src/js/files/components/UploadItem.js

```diff
@@ -26,16 +27,51 @@ const UploadItemTitle = styled.div`
     }
     span:last-child {
         margin-left: auto;
+        color: ${props => (props.failed ? getColor({ theme: props.theme, color: "red" }) : "inherit")};
+        font-size: ${props => (props.failed ? getFontSize("sm") : "inherit")};
+        font-style: ${props => (props.failed ? "italic" : "normal")};
+    }
+    i.fa-times {
+        color: ${props => getColor({ theme: props.theme, color: "red" })};
+        font-size: 20px;
+    }
+    i.fa-trash {
+        margin-left: 5px;
+        font-size: 14px;
     }
 `;
 
-export const UploadItem = ({ name, progress, size }) => (
-    <StyledUploadItem>
-        <AffixedProgressBar now={progress} />
-        <UploadItemTitle>
-            {progress === 100 ? <Loader size="14px" /> : <Icon name="upload" />}
-            <span>{name}</span>
-            <span>{byteSize(size)}</span>
-        </UploadItemTitle>
-    </StyledUploadItem>
-);
+export const UploadItem = ({ name, progress, size, failed, localId, onRemove }) => {
+    let [now, progressColor] = [progress, "blue"];
+    let uploadIcon = progress === 100 ? <Loader size="14px" /> : <Icon name="upload" />;
+    let uploadBookend = byteSize(size);
+
+    if (failed) {
+        [now, progressColor] = [100, "red"];
+        uploadIcon = <Icon name="times" />;
```

Comment:
Use `color` prop of `Icon` instead of styling `i.fa-times`. Is that not working?

Reply:
It works, but when colored by passing props to the `Icon` component the icon will change colors when hovered. Given that the the`fa-times` icon is not interactable I didn't think having a recolor on hover would make sense. I could instead modify the base `Icon` behavior to do something like only recolor on hover if there is a passed function, but that could easily have side effects elsewhere.

Reply:
Alright fixed. I updated the icon color change to to recolor if either a) it is passed an onClick or b) hoverable is passed. On a quick inspection, I wasn't able to find any examples in the UI where this caused a button to not change color when it should have.